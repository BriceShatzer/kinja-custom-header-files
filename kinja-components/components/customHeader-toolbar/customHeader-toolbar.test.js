import * as React from 'react';
import { shallow } from 'enzyme';

import  CustomHeaderToolbar from './customHeader-toolbar';

describe('<CustomHeaderToolbar />', () => {
	it('should render one button in closed mode', () => {
		const wrapper = shallow(<CustomHeaderToolbar isEditMode={false} toggleHandler={() => {}} saveHandler={() => {}} />);
		expect(wrapper).toMatchSnapshot();
	});
	it('should render one button in edit mode', () => {
		const wrapper = shallow(<CustomHeaderToolbar isEditMode={true} toggleHandler={() => {}} saveHandler={() => {}} />);
		expect(wrapper).toMatchSnapshot();
	});
});
