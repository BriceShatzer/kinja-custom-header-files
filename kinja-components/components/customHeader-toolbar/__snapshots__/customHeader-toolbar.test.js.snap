// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CustomHeaderToolbar /> should render one button in closed mode 1`] = `
<customHeader-toolbar__CustomHeaderToolbarWrapper>
  <OpenButton
    onClick={[Function]}
  />
</customHeader-toolbar__CustomHeaderToolbarWrapper>
`;

exports[`<CustomHeaderToolbar /> should render one button in edit mode 1`] = `
<customHeader-toolbar__CustomHeaderToolbarWrapper>
  <SaveButton
    onClick={[Function]}
  />
  <CloseButton
    onClick={[Function]}
  />
</customHeader-toolbar__CustomHeaderToolbarWrapper>
`;
